CloudWorkshop_DIPC_200.md

```diff
@@ -24,23 +24,23 @@ Your will need:
 ### Login into DIPC using Oracle Cloud Services Dashboard
 
 1. In your web browser, navigate to cloud.oracle.com, then click Sign in.
-2. Provide the cloud account: oscnas001 then **\<Enter\>**
-3. Provide your user name and password, then click "Sign In" button. You will land in the Dasboard screen ![](images/200/image200_1.png)
-4. In the "Data Integration Platform Cloud" service box click on the hamburguer menu and then select "Open Service Console" ![](images/200/image200_2.png)
-5. Click on the hamburger menu of the DIPC server assigned to you, then click "Data Integration Platform Console" ![](images/200/image200_3.png)
-You will be navigated to your DIPC server Home page. ![](images/200/image200_4.png)
+2. Provide the cloud account: for example XXXXXXX then **\<Enter\>**.
+3. Provide your user name and password, then click "Sign In" button. You will land in the Dasboard screen. ![](images/200/image200_1.png)
+4. Select the hamburger menu button on the top left corner. Locate "Data Integration" in your list of services, then select the service. ![](images/200/image200_2p.png)
+5. Click on the right-hand-side hamburger menu of the DIPC instance assigned to you, then click the first option to open the console. ![](images/200/image200_3p.png)
+You will be navigated to your DIPC instance web console home page. ![](images/200/image200_4p.png)
 
 ### Login into DIPC using direct URL
 
-1. Open a browser window an provide your DIPC server URL. The URL will be provided by the instructor and will look like this one "https://osc132657dipc-oscnas001.uscom-central-1.oraclecloud.com/dicloud"
-2. Provide your user name and password, then click "Sign In" button ![](images/200/image200_1.png)
-You will be navigated to your DIPC server Home page.
+1. Open a browser window an provide your DIPC server URL. The URL will be provided by the instructor and will look something like this one "https://dipc01-orasenatdpltintegration02.adipc.ocp.oraclecloud.com/dicloud".
```

Comment:
Make this change when we get our OSC environment. The URL will be much closer to what they will get during the workshop